doc/tests/protocols/plant_search.md

```diff
@@ -0,0 +1,45 @@
+# Plant Search Test Protocol
```

Comment:
Ah no sorry, just move this file into manual.

Reply:
And have a look here to be 100% sure that it's the same structure (https://github.com/ElektraInitiative/PermaplanT/tree/master/doc/tests/manual)

Reply:
Yup git did not the delete the old files during the merge apparently.
Is should now be fixed.